benches/bench.rs

```diff
@@ -626,11 +630,17 @@ fn benchmark_complex(c: &mut Criterion) {
                         let output_lines_200 = output_lines_200.await.len();
                         let output_lines_404 = output_lines_404.await.len();
 
+                        // binomial distribution
+                        let sample_stdev = (output_lines_all as f64
```

Comment:
Could you use `debug_assert` for the test runs, so that they don't affect measurement in release? https://doc.rust-lang.org/std/macro.debug_assert.html